apps/oidc-provider-nest/src/environments/environment.dev.ts

```diff
@@ -6,3 +6,4 @@ export const environment = {
 };
 
 export { devDbConfig as dbConfig } from './ormconfig';
+export { ADMIN_DEFAULTS as ADMIN_DEFAULTS } from './secrets';
```

Comment:
Minor point: No need to alias the export with the same name.

Reply:
Fixed.